docs/proposals/pod-resource-management.md

```diff
@@ -0,0 +1,394 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# Pod level resource management in Kubelet
+
+**Author**: Buddha Prakash (@dubstack), Vishnu Kannan (@vishh)
+
+**Last Updated**: 06/23/2016
+
+**Status**: Draft Proposal (WIP)
+
+This document proposes a design for introducing pod level resource accounting to Kubernetes, and outlines the implementation and rollout plan.
+
+<!-- BEGIN MUNGE: GENERATED_TOC -->
+
+- [Pod level resource management in Kubelet](#pod-level-resource-management-in-kubelet)
+  - [Introduction](#introduction)
+  - [Non Goals](#non-goals)
+  - [Motivations](#motivations)
+  - [Design](#design)
+    - [Proposed cgroup hierarchy:](#proposed-cgroup-hierarchy)
+      - [Quality of Service classes](#quality-of-service-classes)
+      - [Guaranteed](#guaranteed)
+      - [Burstable](#burstable)
+      - [Best Effort](#best-effort)
+    - [With Systemd](#with-systemd)
+    - [Hierarchy Outline](#hierarchy-outline)
+  - [Implementation Plan](#implementation-plan)
+      - [Top level Cgroups for QoS tiers](#top-level-cgroups-for-qos-tiers)
+      - [Pod level Cgroup creation and deletion (Docker runtime)](#pod-level-cgroup-creation-and-deletion-docker-runtime)
+      - [Container level cgroups](#container-level-cgroups)
+      - [Rkt runtime](#rkt-runtime)
+      - [Add Pod level metrics to Kubelet's metrics provider](#add-pod-level-metrics-to-kubelets-metrics-provider)
+  - [Rollout Plan](#rollout-plan)
+
+<!-- END MUNGE: GENERATED_TOC -->
+
+## Introduction
+
+As of now [Quality of Service](../../docs/design/resource-qos.md) is not enforced at a pod level. Excepting pod evictions, all the other QoS features are not applicable at the pod level.
```

Comment:
As of now Quality of Service (QoS) is not enforced at a pod level.

(define the acronym at first use, and then use it throughout this document consistently)